### PR TITLE
Get endpoint for profile

### PR DESCRIPTION
**Pull request by dev-danny87**

### CODE REVIEW

File: src/modules/profile/profile.controller.ts [hunk 1]
```diff
@@ -13,4 +20,16 @@ export class ProfileController {
...
+  @Get()
+  @UseGuards(AuthGuard('jwt'))
+  async getProfile(@Request() req) {
+    const user = req.user;
+    const profile = await this.profileService.findOne(user.id);
+    if (profile === undefined || profile === null) {
+      return { message: 'Profile not found' };
+    } else {
+      return profile;
+    }
```
Comment (Rayandson) [inline src/modules/profile/profile.controller.ts:33]: Again, let's keep these checks or validations in service, and you can throw an error 404 when a profile is not found. I recommend using the CustomNotFoundException to keep a pattern in the application.

File: src/modules/profile/profile.service.ts [hunk 2]
```diff
@@ -41,4 +41,8 @@ export class ProfileService {
 
     return { profileId: response.identifiers[0].id };
   }
+
+  async findOne(userId: number) {
+    return this.profileRepository.findOne({ where: { userId } });
+  }
```
Comment (Rayandson) [inline src/modules/profile/profile.service.ts:47]: Since there is a findOneByUserId method in the repository, you call it here.

**Review by Rayandson — CHANGES_REQUESTED** [review]

File: src/modules/profile/profile.service.ts (hunk 2 above)
Comment (dev-danny87) [reply src/modules/profile/profile.service.ts:47]: Nice said, done.

File: src/modules/profile/profile.controller.ts (hunk 1 above)
Comment (dev-danny87) [reply src/modules/profile/profile.controller.ts]: done.

**Review by Rayandson — APPROVED** [review]

**Pull request merged**